src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -223,54 +258,64 @@ salmon_rnaseq_10x:
 #    alt-names: [salmon_rnaseq_10x]
     alt-names: []
     primary: false
+    contains-pii: false
     vitessce-hints: ['is_sc', 'rna']
 
 sciATACseq:
     description: sciATAC-seq
     alt-names: []
     primary: true
+    contains-pii: true
     vitessce-hints: []
 
 sc_atac_seq_sci:
     description: sciATAC-seq [SnapATAC]
```

Comment:
Same complaint as above about this key name not distinguishing from raw protected data...

Reply:
That one is primary: true, meaning that it is raw data, and contains-pii: true, meaning that it does need to be protected.

Reply:
I'm referring to `sc_atac_seq_sci` which is `primary:false`. But, I agree that we can fix these keys later.